### ISSUE TITLE
The Unity project reported an error when introducing the compiled neoutils.framework.

### ISSUE THREAD

**Issue by chrischen1021213933** [O3Labs/neo-utils#50]
There is no problem with neoutils.framework for use in OC and Swift projects. Introduced in the project written in unity language, the compilation is successful but the runtime reports an error. The error message is "runtime/cgo: could not obtain pthread_keys tried xxx". The content screenshot is as follows. 
![unity_error_msg](https://user-images.githubusercontent.com/22274258/62449186-3e543d00-b79c-11e9-8670-f18b1b258279.png)
I have checked the related documentation but have not found a solution.  Thank you very much for helping to solve this problem as soon as possible.

**Comment by WyattMufson** [comment]
I've had no issues linking https://github.com/Ryucoin/neovm-utils to Unity projects for iOS

**Comment by chrischen1021213933** [comment]
Thank you for your reply. But this error does happen in the current Test Unity project, and no other SDKs are used in the project. When I delete the SDK -> Link Binary With Libraries in Build Phases and delete the relevant code, the test project will run normally. How can I fix this error that description as "runtime / cgo: unable to get pthread_keys", is there a way to fix it?

**Comment by chrischen1021213933** [comment]
@apisit I hope you can help me solve this problem, thank you

**Comment by saltyskip** [comment]
Can you share this sample project on github?

**Comment by WyattMufson** [comment]
You might need to use the build script in https://github.com/O3Labs/neo-utils/pull/45 to generate the framework? Additionally, as mentioned in the neovm-utils repo "The compiled headers created by gomobile should be wrapped in NS_ASSUME_NONNULL macros until this is resolved by gomobile (https://stackoverflow.com/a/36164132/3830876)."

**Comment by WyattMufson** [comment]
@saltyskip @apisit #45 should be merged into master. You'll be able to use neo-utils in iOS apps without disabling bitcode (which will eventually be required by Apple).

**Comment by chrischen1021213933** [comment]
[https://jmwu.oss-cn-beijing.aliyuncs.com/SDKXCodeProject.zip](url)
Hello, I uploaded the relevant code, you can click the link to download. I import DappbirdsSDK.framework in the UnityIOSCommunity.m file, and neoutils.framework is used in DappbirdsSDK, so this error occurs. 
You can delete DappbirdsSDK.frameWork in this project and just introduce neoutils using #import <neoutils/neoutils.h>. The error in the screenshot above will still appear when the project is running in the Simulator.

**Comment by chrischen1021213933** [comment]
Sorry, you need to copy this link: https://jmwu.oss-cn-beijing.aliyuncs.com/SDKXCodeProject.zip and open it in your browser.

**Comment by chrischen1021213933** [comment]
@apisit @WyattMufson @saltyskip

**Comment by apisit** [comment]
I can't download your project via the link.  can you put it on github?

**Comment by chrischen1021213933** [comment]
Because the project is too large to upload to github, please provide your email address and I will send it to your email address. Thank you.

**Comment by apisit** [comment]
I believe github can handle a large project just fine. also I don't think email can handle large attachment. 
how big is the project?

**Comment by WyattMufson** [comment]
I've had no problem putting Unity projects on Github

**Comment by chrischen1021213933** [comment]
https://github.com/raisezhang/UI-Tools/blob/master/SDKXCodeProject.zip
We have uploaded to GitHub, this is the link address, I imported neoutils in the UnityIOSCommunity.m file, the compiler will not report an error, but will report an error when running the emulator.

**Comment by chrischen1021213933** [comment]
Sorry to bother you, we urgently need to fix this problem. Can you find the reason for the error, or have any other suggestions?
@WyattMufson @apisit

**Comment by apisit** [comment]
@chrischen1021213933 I downloaded your project and run it locally. Have the same error.
I tried to edit Architecture settings in Build Settings as I see that you have not set Build architecture correctly but still get the same error. Do you have empty ios + unity project?

**Comment by chrischen1021213933** [comment]
I think we have solved this problem. The reason for this error is that this iOS project is a version of the simulator compiled by Unity. We found an introduction to go mobile on github at [https://github.com/golang/go/wiki/Mobile](url), The iOS Simulator module for this page describes the go project with errors in the iOS simulator.
So we recompile the version running on the iPhone, the project works fine, and no relevant error messages appear.
thank you very much.
